source/includes/harvest/_approvals.md

```diff
@@ -0,0 +1,192 @@
+| sequential | true or false based on whether the groups in this flow must approve sequentially |
+| version | The specific version of this flow. |
+| approval_type | one of 'offer_job', 'open_job', or 'offer_candidate' |
+| approval_status | one of 'pending', 'approved', or 'rejected' |
+| job_id | The job for which approval is configured. This always has a value. |
+| requested_by_user_id | The user who requested this approval be started. |
+
+## GET: List Approvals For Job
+
+```shell
+curl 'https://harvest.greenhouse.io/v1/jobs/{id}/approval_flows'
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+```json
+[
+    {
+        "id": 49394,
+        "offer_id": 45545,
+        "sequential": true,
+        "version": 2,
+        "approval_type": "offer_candidate",
+        "approval_status": "pending",
+        "job_id": 12321,
+        "requested_by_user_id": 543
+    },
+    {
+        "id": 49395,
+        "offer_id": nil,
+        "sequential": true,
+        "version": 1,
+        "approval_type": "offer_job",
+        "approval_status": "pending",
+        "job_id": 12321,
+        "requested_by_user_id": 545
+    },
+]
+```
+
+List all of a job's approval flows
+
+### HTTP Request
+
+`GET https://harvest.greenhouse.io/v1/jobs/{id}/approval_flow`
+
+### Querystring parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| id | The id of the job for which you'd like to see approvals |
+
+<br>
+[See noteworthy response attributes.] (#the-approval-flow-object)
+
+## GET: Retrieve Approval Flow
+
+```shell
+curl 'https://harvest.greenhouse.io/v1/approval_flows/{id}'
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+```json
+{
+  "id": 49394,
+  "offer_id": 45545,
+  "sequential": true,
+  "version": 2,
+  "approval_type": "offer_candidate",
+  "approval_status": "pending",
+  "job_id": 12321,
+  "requested_by_user_id": 543
+}
+```
+
+### HTTP Request
+
+`GET https://harvest.greenhouse.io/v1/approval_flows/{id}`
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+id | The ID of the approval flow to retrieve
+
+<br>
+[See noteworthy response attributes.] (#the-approval-flow-object)
+
+## POST: Request Approvals
+
+```shell
+curl -X POST 'https://harvest.greenhouse.io/v1/approval_flows/{id}/request_approvals'
+-H "On-Behalf-Of: {greenhouse user ID}"
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+Given an approval flow, request it be started. This will change the state of the first approver group from "waiting" to "due" and mark the On-Behalf-Of user as the requesting user. This endpoint will fail with a 403 response if the On-Behalf-Of user can't see the hiring plan or can't request approvals. It will fail with a 422 response if the approval flow can't be started.
+
+### HTTP Request
+
+`POST https://harvest.greenhouse.io/v1/approval_flows/{id}/request_approvals`
+
+## GET: Pending Approvals
```

Comment:
Since the rest of these use the job id, would it make sense to call this one GET: User's Pending Approvals?

Reply:
Clarified this. Also added it to the keyset paging list.